### PR TITLE
Add styles prop to TimeseriesSearch

### LINKED ISSUE #209: Add styles prop to TimeseriesSearch

### PR DESCRIPTION
**Pull request by follan**
Fixes #209

### CODE REVIEW

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 1]
```diff
@@ -258,7 +263,12 @@ export class TimeseriesSearch extends React.Component<
           onSearch={this.fetchTimeseries}
           strings={{ searchPlaceholder: 'Search for timeseries' }}
         />
-        <TagList style={{ marginTop: '8px' }}>
+        <TagList
+          style={{
+            ...(styles && styles.list ? styles.list : {}),
+            marginTop: '8px',
```
Comment (xitman78) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: How it will be possible to override `marginTop`?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-210.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 2]
```diff
@@ -265,8 +265,8 @@ export class TimeseriesSearch extends React.Component<
         />
         <TagList
           style={{
-            ...(styles && styles.list ? styles.list : {}),
             marginTop: '8px',
```
Comment (xitman78) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Why not to move `marginTop: '8px',` to `TagList` defined with styled components?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-210.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx (hunk 2 above)
Comment (follan) [reply src/components/TimeseriesSearch/TimeseriesSearch.tsx]: That's a lot better. `TagList`already sets `marginTop: '16px'`, which is instantly overridden

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-210.gearboxjs.preview.cogniteapp.com

**Pull request merged**